components/speechbubble/Speechbubble.tsx

```diff
@@ -0,0 +1,14 @@
+import styles from "./Speechbubble.module.css";
+
+export default function Speechbubble() {
+  return (
+    <>
+      <div className={styles.bubble}>
+        <p className={styles.text}>
+          Hello (Name)! Nice to meet you! I am Albert and i love to travel!
+          Would you come along with me through Australia?
```

Comment:
Maybe use "Will you come along with me" or "Would you like to come along with me", the conjugal form sounds not right here. 🐼

Reply:
I agree!